lib/mayaUsd/render/mayaToHydra/renderGlobals.cpp

```diff
@@ -42,162 +44,329 @@ TF_DEFINE_PRIVATE_TOKENS(
+    mtohRenderOverride_AddAttribute("mtoh", "Show Wireframe on Selected Objects", "mtohWireframeSelectionHighlight", $fromAE);
+    mtohRenderOverride_AddAttribute("mtoh", "Highlight Selected Objects", "mtohColorSelectionHighlight", $fromAE);
+    mtohRenderOverride_AddAttribute("mtoh", "Highlight Color for Selected Objects", "mtohColorSelectionHighlightColor", $fromAE);
+)mel"
+#if USD_VERSION_NUM >= 2005
+R"mel(
+    mtohRenderOverride_AddAttribute("mtoh", "Highlight outline (in pixels, 0 to disable)", "mtohSelectionOutline", $fromAE);
+)mel"
+#endif
+#if USD_VERSION_NUM > 1911 && USD_VERSION_NUM <= 2005
+R"mel(
+    mtohRenderOverride_AddAttribute("mtoh", "Enable color quantization", "mtohColorQuantization", $fromAE);
+)mel"
+#endif
+
+R"mel(
+}
+
+global proc mtohRenderOverride_AEAttributesCallback(string $nodeName) {
+    if (`nodeType $nodeName` != "renderGlobals")
+        return;
+
+    editorTemplate -beginLayout "Hydra Settings" -collapse 1;
+        mtohRenderOverride_AddMTOHAttributes(1);
+        for ($renderer in `mtoh -lr`) {
+            string $displayName = `mtoh -getRendererDisplayName -r $renderer`;
+            editorTemplate -beginLayout $displayName -collapse 1;
+                string $optionsCmd = "mtohRenderOverride_" + $renderer + "Options(1);";
+                eval($optionsCmd);
+            editorTemplate -endLayout;
+        }
+    editorTemplate -endLayout;
+}
+
+// Make our attributes look nice and get sent from the AttributeEditor
+callbacks -o mtoh -hook AETemplateCustomContent -addCallback mtohRenderOverride_AEAttributesCallback;
+)mel";
+
+constexpr auto _renderOverrideOptionBoxTemplate = R"mel(
+global proc {{override}}OptionBox() {
+    string $windowName = "{{override}}OptionsWindow";
+    if (`window -exists $windowName`) {
+        showWindow $windowName;
+        return;
+    }
+
+    // XXX: Could have an optionVar controlling -userDefaults flag
+    //
+    mtoh -createRenderGlobals -r "{{hydraplugin}}" -userDefaults;
+
+    window -title "Maya to Hydra Settings" "{{override}}OptionsWindow";
+    scrollLayout;
+    frameLayout -label "Hydra Settings";
+    columnLayout;
+    mtohRenderOverride_AddMTOHAttributes(0);
+    setParent ..;
+    setParent ..;
+
+    frameLayout -label "{{hydraDisplayName}}" -collapsable true;
+    columnLayout;
+    {{override}}Options(0);
+    setParent ..;
+    setParent ..;
+
+    setParent ..;
+
+    showWindow $windowName;
+}
+)mel";
+
+
+static constexpr const char* kMtohNSToken = "_mtohns_";
+static const std::string kMtohRenderePostFix("__");
+
+static MString _MangleColorAttribute(const MString& attrName, unsigned i) {
+    static const MString kMtohCmptToken("_mtohc_");
+    static const std::array<MString, 4> kColorComponents = { "R", "G", "B", "A" };
+    if (i < kColorComponents.size())
+        return attrName + kMtohCmptToken + kColorComponents[i];
```

Comment:
Another braceless-if

Now that we've decided how to replace these, there are a number of them still in the code - I found that if you just search for this regexp:

`if *\([^\n{]+\n`

...it should find them (as well as a few multi-line-if red-herrings, but not many).